src/schema/objects/entities.yaml

```diff
@@ -81,23 +36,25 @@ ceagent:
     with the same label.
   type: string
   format: label
-tracer:
-  name: Tracer
-  entity: trc
+density:
+  name: Density
+  entity: den
   description: |
-    The `trc-<label>` key/value can be used to distinguish
-    sequences using different tracers.
-    The key `TracerName` MUST also be included in the associated JSON file,
-    although the label may be different.
+    Density of non-parametric surfaces.
+    MUST have a corresponding `Density` metadata field to provide
+    interpretation.
+
+    This entity is only applicable to derivative data.
```

Comment:
Side note: I suspect you have thought about this, but we might need at some point to create a rule to keep track of derivatives only entities.

Reply:
You're right. We can always add whatever fields we want, but it's probably something that can be formalized with the changes proposed by @erdalkaraca in #884.